node.go

```diff
@@ -36,6 +36,10 @@ type Node struct {
 	// Async handler
 	handler Handler
 
+	// Used to chain go routines posting events to async handler.
+	nextSCCh  chan bool
```

Comment:
Took me awhile to figure that out.. ;) That is why I suggested something more descriptive but still short. Maybe just stateChg?